views/empty_state.erb

```diff
@@ -1,10 +1,15 @@
 <%
-  empty_state = Entities::Pages.page(request.path_info).empty_state
+  page = Entities::Pages.page(request.path_info)
+  empty_state = page.empty_state
 %>
 
 <section class="empty-state-container">
   <div class="empty-state-content owl">
-    <h3><%=empty_state.heading%></h3>
+    <% if page.parent.nil? %>
+      <h2 class="h3"><%=empty_state.heading%></h2>
+    <% else %>
+      <h3><%=empty_state.heading%></h3>
+    <% end %>
```

Comment:
There is similar logic in here so this is where I'd put the logic: https://github.com/mlibrary/patron_account/blob/main/lib/entities/empty_state.rb

maybe add a `heading-tag` and `heading-class` attribute in there. Then you can do `< <%=empty_state.heading-tag%> class="<%=empty_state.heading-class%>">`